WebContent/js/containers/ContentViewer.js

```diff
@@ -135,7 +135,7 @@ export class ContentViewer extends React.Component {
         }
     }
 
-    handleCloseAllExceptTabs(index) {
+    handleCloseOtherTabs(index) {
```

Comment:
I don't think these Close All changes were intended

Reply:
I just changed the function names and Context Menu option names to more user friendly